java/daikon/PptTopLevel.java

```diff
@@ -2680,6 +2681,7 @@ public void addConditions(Splitter[] splits) {
   }
 
   /*@Pure*/
+  // TODO: Why is this method synchronized? There is no synchronization happening when proverStack is initialized.
```

Comment:
I agree it's unnecessary.  Please remove the synchronization, and note it as an anecdote to report about finding a bug due to the process of annotating with the Lock Checker.